services/merge.ts

```diff
@@ -0,0 +1,183 @@
+import { sparqlEscapeUri } from 'mu';
+import { updateSudo } from '@lblod/mu-auth-sudo';
+
+import { CustomError } from '../utils/custom-error';
+import { Conflict } from '../types';
+
+export async function addIsConflictingFlagToPersons(
+  conflicts: Array<Conflict>,
+): Promise<void> {
+  if (conflicts.length === 0) {
+    return;
+  }
+
+  const values = conflicts.map(
+    (conflict) =>
+      `( ${sparqlEscapeUri(conflict.conflictUri)} ${sparqlEscapeUri(conflict.personUri)})`,
+  );
+  const queryString = `
+    PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+    PREFIX person: <http://www.w3.org/ns/person#>
+    PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+    PREFIX dct: <http://purl.org/dc/terms/>
+
+    DELETE {
+      GRAPH ?g {
+        ?conflict dct:modified ?modified .
+      }
+    }
+    INSERT {
+      GRAPH ?g {
+        ?conflict ext:conflictsWith ?person  .
+        ?conflict dct:modified ?modified .
+      }
+    }
+    WHERE {
+      VALUES ( ?conflict ?person) { ${values.join('\n')} }
+      GRAPH ?g {
+        ?conflict a person:Person .
+
+        OPTIONAL {
+          ?conflict dct:modified ?modified .
+        }
+      }
+      ?g ext:ownedBy ?organization .
+      BIND(NOW() AS ?now)
+    }
+  `;
+
+  try {
+    await updateSudo(queryString);
+  } catch (error) {
+    throw new CustomError(
+      `Something went wrong while adding ext:conflictsWith flag to ${conflicts.length} persons.`,
+    );
+  }
+}
+
+export async function updateConflictUsageToPersonAsSubject(
+  conflicts: Array<Conflict>,
+): Promise<void> {
+  if (conflicts.length === 0) {
+    return;
+  }
+
+  const values = conflicts.map(
+    (c) =>
+      `( ${sparqlEscapeUri(c.conflictUri)} ${sparqlEscapeUri(c.personUri)} )`,
+  );
+  const queryString = `
+    PREFIX persoon: <http://data.vlaanderen.be/ns/persoon#>
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+    PREFIX adms: <http://www.w3.org/ns/adms#>
+    PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+    PREFIX dct: <http://purl.org/dc/terms/>
+    PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+    PREFIX person: <http://www.w3.org/ns/person#>
+
+    DELETE{
+      GRAPH ?conflictG {
+        ?conflict dct:modified ?modified .
+      }
+    }
+    INSERT{
+      GRAPH ?conflictG {
+        ?person ?p ?o .
+        ?person mu:uuid ?id .
+        ?person dct:modified ?now .
+        ?person adms:identifier ?identifierP .
+        ?identifierP ?ip ?io .
+        ?person persoon:heeftGeboorte ?geboorteP .
+        ?geboorteP ?gp ?go .
+      }
+    }
+    WHERE {
+      VALUES ( ?conflict ?person ) {
+        ${values.join('\n')}
+      } 
+      GRAPH ?g {
+        ?person a person:Person .
+        ?person mu:uuid ?id .
+        
+        OPTIONAL {
```

Comment:
because we know this is a resolvable conflict, it means that the identifier and birth date must be set SO these don't need to be optional.